tests/test_service.py

```diff
@@ -830,3 +830,183 @@ def testGetAlertWithMultipleTriggers(self):
             self.assertEquals(len(alert.triggers), 2)
         self.assertEquals(alert.triggers[100].argus_id, 100)
         self.assertEquals(alert.triggers[101].argus_id, 101)
+
+
+
+class TestCompAlert(TestServiceBase):
+
+    @mock.patch('requests.Session.post', return_value=MockResponse(json.dumps(compalert_D), 200))
+    def testAddCompAlert(self, mockPost):
+        alert = Alert.from_dict(compalert_D)
+        self.assertTrue(isinstance(alert, Alert))
+        delattr(alert, "id")
+        res = self.argus.alerts.create_composite_alert(alert)
+        self.assertTrue(isinstance(res, Alert))
```

Comment:
These lines are duplicated in several methods. Can you pull it into 1 method?

Reply:
Fixed the code, consolidated all the common functionality into a single method